function-reference/query-functions/text.xml

```diff
@@ -23,7 +23,7 @@ query variables and similar state.
       <c>use flagname</c>
     </ti>
     <ti>
-      Returns a true value if and only if <c>flagname</c> is enabled.
+      Returns a true value only if <c>flagname</c> is enabled.
```

Comment:
Or we could follow the PMS wording, to make it even clearer:
"Returns a true value if `flagname` is enabled, false otherwise."

Reply:
I like that too.